### PR TITLE
Fix Textmate grammar provider scope collision

### LINKED ISSUE #15: Textmate grammar provider scope collision when mixed with @theia/json
I noticed the following exception when starting a Theia application that includes `@theia/plantuml` and `@theia/json` :

```
root WARN Error: a registered grammar provider for 'source.json' scope is overridden
    at e.registerTextmateGrammarScope (http://localhost:3003/bundle.js:19:422576)
    at e.registerTextmateLanguage (http://localhost:3003/bundle.js:117:1008834)
    at e.initialize (http://localhost:3003/bundle.js:30:501976)
    at e.<anonymous> (http://localhost:3003/bundle.js:12:365064)
    at http://localhost:3003/bundle.js:12:360054
    at Object.next (http://localhost:3003/bundle.js:12:360159)
    at http://localhost:3003/bundle.js:12:359072
    at new Promise (<anonymous>)
    at a (http://localhost:3003/bundle.js:12:358849)
    at e.startContributions (http://localhost:3003/bundle.js:12:364835)

```

It seems that the collision is due to the reuse of the same scope id/key: `source.json`: 
https://github.com/theia-ide/theia-plantuml-extension/blob/master/plantuml/src/browser/plantuml-grammar-contribution.ts#L58

I guess this should rather be something like ~`source.plantuml`?~ update: actually `source.wsd` is used in the grammar file. 

Note: since `@theia/debug` depends on `@theia/json`, the problem can happen even if the later is not included directly.

### PR DESCRIPTION
**Pull request by marcdumais-work**
when mixed with @theia/json, in the same Theia application.

Fixes #15

### CODE REVIEW

File: plantuml/src/browser/plantuml-grammar-contribution.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const PLANTUML_LANGUAGE_NAME = 'PlantUML';
 
 @injectable()
 export class PlantumlGrammarContribution implements LanguageGrammarDefinitionContribution {
+    readonly scopeName = 'source.wsd';
```
Comment (AlexTugarev3) [inline plantuml/src/browser/plantuml-grammar-contribution.ts]: `source.plantuml` would be better to void another potential collision.

Comment (AlexTugarev) [inline plantuml/src/browser/plantuml-grammar-contribution.ts:16]: ~`source.plantuml` would be better to void another potential collision.~


nvm, I read your comment.

**Review by AlexTugarev — APPROVED** [review]
Thanks @marcdumais-work!

File: plantuml/src/browser/plantuml-grammar-contribution.ts (hunk 1 above)
Comment (AlexTugarev3) [reply plantuml/src/browser/plantuml-grammar-contribution.ts]: nvm, I read your comment.

**Pull request merged**